assets/src/edit-story/components/keyboard/index.js

```diff
@@ -46,13 +46,15 @@ const globalRef = createRef();
  *
  * @param {{current: Node}} ref
  * @param {string|Array|Object} keyNameOrSpec
+ * @param {string|undefined} type
  * @param {function(KeyboardEvent)} callback
  * @param {Array|undefined} deps
  */
-export function useKeyDownEffect(
+function useKeyEffect(
   ref,
   keyNameOrSpec,
-  callback,
+  type = 'keydown',
```

Comment:
IMHO, default value for this is not needed. It's odd to require `callback` default to enable this.